skopt/__init__.py

```diff
@@ -76,9 +76,6 @@ def f(x):
            "benchmarks",
            "callbacks",
            "learning",
+           "optimizer",
            "plots",
-           "space",
-           "gp_minimize",
-           "dummy_minimize",
-           "forest_minimize",
-           "gbrt_minimize")
+           "space")
```

Comment:
Yes, I'll fix that. But I think `__all__` affects only what gets imported during `from skopt import *`. `from skopt import gp_minimize` should still work since I"ve imported from `optimizer` above. (http://stackoverflow.com/a/64130/1170730)